lib/broadside/configuration/deploy.rb

```diff
@@ -22,14 +22,18 @@ class DeployConfig < ConfigStruct
         :env_vars,
```

Comment:
it's really hard to jump into a project and change critical code without specs!  i would feel like 1000% more confident about this PR if running rspec actually confirmed the right calls were made to AWS.